content/types/_stats.textile

```diff
@@ -8,14 +8,19 @@ h4.
   ruby:    Attributes
   python:  Keyword arguments
 
+- <span lang="default">intervalId</span><span lang="ruby,python">interval_id</span><span lang="csharp">IntervalId</span> := the time at which the interval covered by this @Stats@ object starts. For example, an interval ID value of "2016-03-01:10:02" represents a single minute interval, in UTC. The granularity does not necessarily indicate the period of time the interval covers — for that you should consult the @unit@; for example an interval ID of "2016-03-01:00" with a @unit@ of @day@ would represent the whole day of the 1st March 2016 (UTC, midnight to midnight), but the same interval ID with a @unit@ of @hour@ would represent only the first hour of that day.<br>__Type: @String@__
```

Comment:
> The granularity does not necessarily indicate the period of time the interval covers

Surely you meant to say "The `intervalId` does not necessarily indicate the period of time the interval covers" ?

I think this is confusing in any case. I think the definition of `unit` would help if it comes before this, and here it should say

> the UTC time at which the time period covered by this @Stats@ object starts. For example, an `intervalId` value of "2018-03-01:10", in a `Stats` object whose `unit` is `day`, would indicate that the period covered is "2018-03-01:10 .. 2018-03-01:11". All `Stats` objects, except those whose `unit` is `minute` have an `intervalId` with resolution of one hour and the time period covered will always begin and end at a UTC hour boundary. For this reason it is not possible to infer the `unit` by looking at the resolution of the `intervalId`. `Stats` objects covering an individual minute will have an `intervalId` indicating that time; for example "2018-03-01:10:02".

Reply:
Updated in 3424176